test.py

```diff
@@ -0,0 +1,6 @@
+import environment.swarmball_env as swarmball
```

Comment:
nie wiem jak sie ten plik tutaj dostał, umie ktos usuwać pliki z commita?

Reply:
git rm plik.py jeśli chcesz wgl usunąć plik z całego projektu. Jeśli chcesz przywrócić plik do stanu "początkowego" czyli takiego który był na gałęzi z której branchowałeś (czyli takiego w którym nie dokonałeś zmiany) to git checkout plik.py